cdap-examples/SportResults/src/main/java/co/cask/cdap/examples/sportresults/SportResults.java

```diff
@@ -37,11 +37,11 @@ public void configure() {
     createDataset("results", PartitionedFileSet.class, PartitionedFileSetProperties.builder()
       // properties for partitioning
       .setPartitioning(Partitioning.builder().addStringField("league").addIntField("season").build())
-        // properties for file set
+      // properties for file set
       .setInputFormat(TextInputFormat.class)
       .setOutputFormat(TextOutputFormat.class)
       .setInputProperty(TextOutputFormat.SEPERATOR, ",")
```

Comment:
Guess this deserves an issue...https://issues.cask.co/browse/CDAP-1757 but I'm not sure how we should handle it. Just document it everywhere we use it?